assets/javascripts/discourse/controllers/user-activity-your-reactions.js.es6

```diff
@@ -0,0 +1,43 @@
+import Controller, { inject as controller } from "@ember/controller";
+import { observes } from "discourse-common/utils/decorators";
+import { action } from "@ember/object";
+import CustomReaction from "../models/discourse-reactions-custom-reaction";
+
+export default Controller.extend({
+  canLoadMore: true,
+  loading: false,
+  application: controller(),
+
+  @action
+  loadMore() {
+    if (!this.canLoadMore) {
+      return;
+    }
+    if (this.loading) {
+      return;
+    }
+    this.set("loading", true);
+    const posts = this.model;
+    if (posts && posts.length) {
+      const beforePostId = posts[posts.length - 1].get("current_user_reaction").id;
```

Comment:
get is not going to save you if posts is not found, eg: null.get(...) doesn’t work